src/hermes.jsx

```diff
@@ -348,6 +351,9 @@ function getSelectedNode () {
 }
 
 function selectionChanged (prevSel, sel) {
+	if (prevSel === null && sel === null) {
```

Comment:
I think they are both initialized to null. It may be that the calling code was causing useless re-renders, but I don't think that is this module's concern.